packages/ERTP/src/paymentLedger.js

```diff
@@ -291,7 +291,12 @@ export const makePaymentLedger = (
    */
   const withdraw = (currentBalance, updatePurseBalance, amount) => {
     amount = coerce(amount);
+    assert(
+      AmountMath.isGTE(currentBalance, amount),
+      X`Withdrawal of ${amount} failed because the purse only contained ${currentBalance}`,
```

Comment:
I wonder whether it's clear which purse "the purse" refers to. I think so... The split between `paymentLedger.js` and `purse.js` is pretty subtle, but I think I get it.

Reply:
I don't think it's clear enough for the folks who didn't manage to put RUN in their fee purse, but that's beyond the scope of ERTP. We have some UI work to do...

Reply:
can we get at the purse's `alleged name`?  It would be a big improvement to tell the user which purse if we can.

Reply:
> I don't think it's clear enough for the folks who didn't manage to put RUN in their fee purse, but that's beyond the scope of ERTP. We have some UI work to do...

Yup, I'm deliberately not closing the original issue for that reason.

Reply:
> can we get at the purse's `alleged name`? It would be a big improvement to tell the user which purse if we can.

I think you might be thinking of the purse's petname in the wallet. There is no alleged name per purse in ERTP. The Far name of a purse is the allegedName of the brand + 'purse', so it only really tells you the brand, which the amount will already do.

Reply:
I wasn't thinking about the petname in the wallet. 

The error message in withdraw is about the balance of a particular purse. Purses have an `allegedName`, which, though not unique, at least indicates the purse's role. It would be helpful to print the name of the offending purse when someone tries to withdraw more than it contains.

Reply:
I think we're talking past each other here. The purse's allegedName would not be helpful, as it is the alleged name of the brand, which is already printed in the error message.